migration/migrator/migrations/course/20210610100802_course_materials_json_to_db.py

```diff
@@ -0,0 +1,121 @@
+"""Migration for a given Submitty course database."""
+
+import json
+from pathlib import Path
+from sqlalchemy import insert
+
+def up(config, database, semester, course):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+    # create tables here
+    database.execute(
+        """
+        CREATE TABLE IF NOT EXISTS course_materials (
+            url TEXT PRIMARY KEY,
```

Comment:
I would suggest changing `url` to `path` or `file_path` since course materials can be links containing only a URL so it might be confusing what value this is supposed to represent 

Unless this is supposed to be the fully formed URL to access the resource from the server